src/ngraph/builder/reduce_ops.cpp

```diff
@@ -111,5 +112,68 @@ namespace ngraph
             return result->add_provenance_group_members_above({value});
         }
 
+        std::shared_ptr<Node> builder::opset1::mean(const Output<Node>& value,
+                                                    const AxisSet& reduction_axes)
+        {
+            const auto xsum = std::make_shared<ngraph::opset1::ReduceSum>(
+                value,
+                ngraph::opset1::Constant::create(
+                    element::i64, Shape{reduction_axes.size()}, reduction_axes.to_vector()),
+                false);
+
+            const auto N = get_num_elements(value.get_shape(), reduction_axes);
+            const auto& et = value.get_element_type();
+
+            const auto divisor = ngraph::opset1::Constant::create(et, Shape{}, {N});
+
+            return std::make_shared<ngraph::opset1::Divide>(xsum, divisor)
+                ->add_provenance_group_members_above({value});
+        }
+
+        std::shared_ptr<Node> builder::opset1::variance(const Output<Node>& value,
+                                                        const AxisSet& reduction_axes,
+                                                        const bool bessel_correction)
+        {
+            std::shared_ptr<Node> mu = mean(value, reduction_axes);
+
+            auto reshape = value.get_shape();
+            for (auto i : reduction_axes)
+            {
+                reshape[i] = 1;
+            }
+
+            mu = std::make_shared<ngraph::opset1::Reshape>(
+                mu,
+                ngraph::opset1::Constant::create(
+                    element::i64,
+                    Shape{reshape.size()},
+                    std::vector<int64_t>(reshape.begin(), reshape.end())),
+                false);
+
+            Output<Node> diff = make_with_numpy_broadcast<ngraph::opset1::Subtract>(value, mu);
```

Comment:
Can you use `opset1::Subtract` without this fancy helper?

Reply:
done